sdk/lib/rtl/threadpool.c

```diff
@@ -19,6 +19,48 @@
  * Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
  */
 
+
+#ifdef __REACTOS__
+#include <rtl_vista.h>
+#define NDEBUG
+#include "wine/list.h"
+#include <debug.h>
+
+#define ERR(fmt, ...)    DPRINT1(fmt, ##__VA_ARGS__)
+#define FIXME(fmt, ...)  DPRINT(fmt, ##__VA_ARGS__)
+#define WARN(fmt, ...)   DPRINT(fmt, ##__VA_ARGS__)
+#define TRACE(fmt, ...)  DPRINT(fmt, ##__VA_ARGS__)
+#ifndef ARRAY_SIZE
+#define ARRAY_SIZE(_x) (sizeof((_x))/sizeof((_x)[0]))
+#endif
+
+/* Winternal ?*/
```

Comment:
/* winternl? */

Reply:
Ah just some notes for me, not relevant for now ill remove the,.